.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# For more information see: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
Is this the interval we use in all projects? `weekly` feels like a much preferred schedule to me

Reply:
Yes, it is what we use in other repositories of the Standard organization, example: `standard/standard`: https://github.com/standard/standard/blob/master/.github/dependabot.yml.

> I would actually personally say that Renovate is better integrated with GitHub and better overall

How true is that ? What is missing in `dependabot` that `Renovate` has ?

I would say it is better to stay consistent with others repos of the org and use `dependabot` instead of `renovabot`.

Reply:
> How true is that ? What is missing in `dependabot` that `Renovate` has ?

Hard to list it all here, check one of my repos:

* https://github.com/voxpelli/eslint-config/pulls
* https://github.com/voxpelli/async-htm-to-string/pulls

And my config:

* https://github.com/voxpelli/renovate-config-voxpelli/blob/3fea9a0cd461d3e9d4eda3c186e83d6d221562d4/package.json#L18

> I would say it is better to stay consistent with others repos of the org and use `dependabot` instead of `renovabot`.

I would generally agree, though the urgency of it is low compared to getting eg. CI to work.

+ if there may be a possibly better alternative, then it might be a good idea to try that out on some repos, and if superior, then move the rest to it, else replace it with the common one.